### ISSUE TITLE
USB host

### ISSUE THREAD

**Issue by comradeFreeman** [MediaDeb/MediaDeb#6]
Доброй ночи! Подскажите, пожалуйста, как перевести USB-порт устройства в режим хоста для подключения периферии, запоминающих  устройств?

**Comment by nekromant** [comment]
Если помню правильно, там где-то в sysfs были ключики, надо смотреть и искать. Но только у моделей, где есть аппаратная поддержка (она была не у всех), и ключики не совсем стандартные. 

На  dwc_otg была опция dr_mode (на новых ядрах ушло в dts), но у медиатеков проде бы не dwc'шный хост был. Больше уже за давностью лет не помню

**Comment by comradeFreeman** [comment]
> Если помню правильно, там где-то в sysfs были ключики, надо смотреть и искать. Но только у моделей, где есть аппаратная поддержка (она была не у всех), и ключики не совсем стандартные.
> 
> На dwc_otg была опция dr_mode (на новых ядрах ушло в dts), но у медиатеков проде бы не dwc'шный хост был. Больше уже за давностью лет не помню

Да, Вы правы насчёт аппаратной поддержки у модели. Открывая запрос, я экспериментировал с THL W11 на прошивке UMI-X2 (с подправленным `scatter`-файлом) и у меня никак не получалось ключами завести режим хоста. Оказалось, что у меня ревизия, которая как раз не поддерживает OTG аппаратно. 

Зато на iOcean X8 вроде как даже режим хоста заработал сам, без манипуляций с системой. Хотя я делал `echo host > /sys/devices/platform/mt_usb/cmode` и `echo host > /sys/devices/platform/mt_usb/portmode`, но после перезагрузки он работает, так что, вероятно, это не влияет. Правда, почему-то трудно работать с носителями: разделы иногда видит не все, после создания оных прямо в системе примонтировать не получается, возможно это проблема ядра или инструментов

**Comment by comradeFreeman** [comment]
Кстати, у iOcean X8 UART **есть**, прикладываю фото и прошу обновить информацию

![IMG_20230328_223620_400](https://user-images.githubusercontent.com/46820095/228756212-f76fa942-6dc5-4031-af36-6ff1fd281881.jpg)
